src/ipsis/course-merge/CourseMergeMergedOfferingCollectionEntity.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * CourseMergeMergedOfferingCollectionEntity class representation of a list of course merge merged offerings as defined in the LMS
+ * See: ISirenCourseMergeSerializer.SerializeMergedCoursesListResult
+ */
+import { Actions, Classes, Rels } from '../../hypermedia-constants.js';
+import { Entity } from '../../es6/Entity.js';
+import { performSirenAction } from '../../es6/SirenAction.js';
+
+export class CourseMergeMergedOfferingCollectionEntity extends Entity {
+	originalSourceCourseMergeOfferings() {
+		return this._entity?.entities?.filter(course => !course.class.includes(Classes.ipsis.sisCourseMerge.originalTarget));
+	}
+
+	originalTargetCourseMergeOffering() {
+		return this._entity?.getSubEntitiesByClass(Classes.ipsis.sisCourseMerge.originalTarget)?.[0];
+	}
+
+	prependOriginalSourceCourseMergeOfferings(previousCourseMergeMergedOfferingCollectionEntity) {
```

Comment:
Not something we have to do, just something to consider:
Could we move all these paging related functions to a `PagedEntity` class and have this and `CourseMergeOfferingCollectionEntity` both extend the `PagedEntity`?

Reply:
I'll keep this in mind for when I add the property for original source courses being deleted out of the LMS.